auto_nag/user_activity.py

```diff
@@ -4,90 +4,97 @@
 
 from enum import Enum
 
-from libmozdata.bugzilla import Bugzilla, BugzillaUser
+from libmozdata import utils as lmdutils
+from libmozdata.bugzilla import BugzillaUser
 
 from auto_nag import utils
+from auto_nag.people import People
 
 DEFAULT_ACTIVITY_WEEKS = 26
+DEFAULT_ABSENT_WEEKS = 13
```

Comment:
> @marco-c if you think that the number of bugs is too high, we could increase the threshold for the `ABSENT` status to reduce the number of bugs.

Yeah, let's start with the same value as activity (so 26 weeks, that is half a year). We can file a follow-up issue to decrease both values in the future.

Reply:
Done in 208a5b14c40f89e940a21628b1ab34ca2e9f2d15, and I filled a follow-up issue  (#1435).